vcs/src/main/java/org/openjdk/skara/vcs/git/GitRepository.java

```diff
@@ -1659,4 +1659,65 @@ public int commitCount() throws IOException {
     public Hash initialHash() {
         return EMPTY_TREE;
     }
+
+    @Override
+    public Optional<Hash> wholeHash(String rev) {
```

Comment:
Isn't this just `::resolve(String)`?

Reply:
What I mean is, you could use resolve instead of introducing this new method.

Reply:
Oh, I don't know why I didn't see some of your review yesterday. Ah, right, I will use resolve and delete the new method